README.md

```diff
@@ -1,78 +1,90 @@
 <p align="center"><a href="http://geomesa.github.io"><img src="https://raw.githubusercontent.com/geomesa/geomesa.github.io/master/img/geomesa-2x.png"></img></a></p>
 
-[![Join the chat at https://gitter.im/locationtech/geomesa](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/locationtech/geomesa?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
-
-### Build Status
-
-Version | Status
------- | -----
-| 1.1.x | [![Build Status](https://api.travis-ci.org/locationtech/geomesa.svg?branch=master)](https://travis-ci.org/locationtech/geomesa)  	| 
-| 1.0.x | [![Build Status](https://travis-ci.org/locationtech/geomesa.svg?branch=accumulo1.5.x%2F1.x)](https://travis-ci.org/locationtech/geomesa)  	| 
-
+<a href="https://gitter.im/locationtech/geomesa?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge" target="_blank"><img src="https://badges.gitter.im/Join%20Chat.svg" alt="Join the chat at https://gitter.im/locationtech/geomesa"></img></a>
```

Comment:
Can you move the gitter link under the splash image (not the logo, the four box image)?  Also, maybe make the four box image a little shorter?